README.md

```diff
@@ -29,8 +29,8 @@
 1. In `config.json`, configure the following values:
     - `account.username`: Your Minecraft account playername.
     - `proxy.whitelist`: Playernames of accounts that are allowed to connect to the proxy.
-2. Run `npm ci --omit=dev`
-3. Run `npm start`
+2. Run `pnpm i`
```

Comment:
```suggestion
2. Run `pnpm i --prod`
```
Don't install dev dependencies.

Reply:
I'm going to hold off on merging until I can test it myself and see if updating the autoeat package resolves this.

Reply:
Additionally, `process.env.pnpm_package_version` is undefined when I run `pnpm run debug-info`.

`process.env.CI` is also undefined when I run `pnpm run test`.

Not sure what could be causing this or where the documentation is for pnpm environment variables.

Reply:
Fixed all the issues mentioned, even auto-eat